addons/website_sale_picking/data/website_sale_picking_data.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="payment_acquirer_onsite" model="payment.acquirer">
+        <field name="name">Pay in store when picking the product</field>
+        <field name="description" type="html">
+            <p>
+                Allows your customers to pay in person, at your shops.
+            </p>
+        </field>
+        <field name="provider">transfer</field>
+        <field name="state">enabled</field>
+        <field name="is_onsite_acquirer">true</field>
+        <field name="redirect_form_view_id" ref="payment_transfer.redirect_form"/>
+        <field name="pending_msg"><![CDATA[
```

Comment:
Don't use CDATA, as the xml structure won't be validated and the content could contain invalid syntax.

Use type="xml" instead :)